index.html

```diff
@@ -1,304 +1,416 @@
-                <p>
-                    A daily selection of privately personalized reads; no accounts or
-                    sign-ups required. Has been the industry's standard.
-                </p>
-                <ul>
-                    <li>
-                        <a href="#">HTML</a>
-                    </li>
-                    <li>
-                        <a href="#">BOOTSTRAP</a>
-                    </li>
-                    <li>
-                        <a href="#">RUBY</a>
-                    </li>
-                </ul>
-                <button type="button">See project</button>
-            </article>
+          <section class="content-desc">
+            <img src="images/icons/pro_sec.svg" alt="Content">
+            <div class="top-left1">
+              <h2>Data Dashboard<br/>Healthcare</h2>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul>
+              <li>
+                <a href="#">HTML</a>
+              </li>
+              <li>
+                <a href="#">BOOTSTRAP</a>
+              </li>
+              <li>
+                <a href="#">RUBY</a>
+              </li>
+            </ul>
+            <button type="button">See project</button>
+          </div>
+          </section>
         </section>
 
         <section class="center-content">
-            <article class="content-desc">
-                <h2>Profesional Art <br />Printing Data</h2>
-                <p>
-                    A daily selection of privately personalized reads; no accounts or
-                    sign-ups required. Has been the industry's standard.
-                </p>
-                <ul>
-                    <li>
-                        <a href="#">HTML</a>
-                    </li>
-                    <li>
-                        <a href="#">BOOTSTRAP</a>
-                    </li>
-                    <li>
-                        <a href="#">RUBY</a>
-                    </li>
-                </ul>
-                <button type="button">See project</button>
-            </article>
+          <section class="content-desc">
+            <img src="images/icons/Data_content_image.svg" alt="">
+            <div class="top-left">
+              <h2>Data Dashboard<br/>Healthcare</h2>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul>
+              <li>
+                <a href="#">HTML</a>
+              </li>
+              <li>
+                <a href="#">BOOTSTRAP</a>
+              </li>
+              <li>
+                <a href="#">RUBY</a>
+              </li>
+            </ul>
+            <button type="button">See project</button>
+          </div>
+          </section>
```

Comment:
- You may have misunderstood the requirement of the previous reviewer. For each card, when you hover over each of the cards, it should look like [this](https://user-images.githubusercontent.com/26597432/236642081-0cd65ff4-f2f4-4f54-add6-be3ae340cfa6.png). [This](https://user-images.githubusercontent.com/26597432/236642091-f843b788-cf58-472e-ad93-ba39e7a603a6.png) is the default look of the cards.